### PR TITLE
Upgrades

### PR DESCRIPTION
**Pull request by grechaw**
This PR updates versions of gradle wrapper, ditches commons-io in favor of Java constructs, and seems to work. 

I also de-tabbified.  I'd like to ask contributors to set their IDEs to use four spaces rather than tabs.

### CODE REVIEW

**Comment by grechaw** [comment]
Addresses #150

**Comment by jmakeig** [comment]
> I'd like to ask contributors to set their IDEs to use four spaces rather than tabs.

Which is why tabs are actually better. (Says the two-space guy.)

File: entity-services-e2e/build.gradle [hunk 1]
```diff
@@ -3,12 +3,18 @@ apply plugin: 'application'
...
 
-dependencies {
-    compile('com.marklogic:data-movement:1.0-SNAPSHOT') {
-        exclude(group: 'com.google.guava')
-    }
-    compile('ch.qos.logback:logback-classic:1.1.7')
-    compile('com.fasterxml.jackson.dataformat:jackson-dataformat-csv:2.7.0')
+repositories { 
+  maven { 
+    url "http://developer.marklogic.com/maven2"
```
Comment (bsrikan) [inline entity-services-e2e/build.gradle]: What is the purpose of adding this internal maven repo @grechaw ?

Comment (jmakeig) [inline entity-services-e2e/build.gradle]: That’s the only place where you can get EA builds. I don’t think we publish these to Maven Central.

Comment (bsrikan) [inline entity-services-e2e/build.gradle]: Understood.

**Pull request merged**